src/services/message.service.ts

```diff
@@ -36,6 +37,7 @@ const isInvalidTokenError = (err: unknown): boolean => (
 const sendMessageToDevice = async (
   deviceToken: string,
   notificationType: string,
+  context: { [key: string]: string },
```

Comment:
I'm not clear on what this is expressing.  Not necessary of course, but...what?

Reply:
That's an [index signature](https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures), which says "the type of `context` is an object with an unknown set of keys (where each is a `string`), with values that are `strings`".

Reply:
very cool, thank you!